policy/modules/contrib/rtorrent.if

```diff
@@ -0,0 +1,111 @@
+## <summary>Policy for rtorrent.</summary>
+
+############################################################
+## <summary>
+##	Role access for rtorrent
+## </summary>
+## <param name="role">
+##	<summary>
+##	Role allowed access
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	User domain for the role
+##	</summary>
+## </param>
+#
+interface(`rtorrent_role',`
+	gen_require(`
+	    attribute_role rtorrent_roles;
+	    type rtorrent_t, rtorrent_exec_t;
+	')
+
+	roleattribute $1 rtorrent_roles;
+
+	# transition from the userdomain to the derived domain
+	domtrans_pattern($2, rtorrent_exec_t, rtorrent_t)
+
+	# allow ps to show rtorrent
+	ps_process_pattern($2, rtorrent_t)
+	allow $2 rtorrent_t:process { signull sigstop signal sigkill };
+
+	ifdef(`hide_broken_symptoms',`
+		#Leaked File Descriptors
+		dontaudit rtorrent_t $2:fifo_file rw_fifo_file_perms;
```

Comment:
This does not make sense as this is allowed by default in the `domtrans_pattern()` here: https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/misc_patterns.spt#L37

That is, without the "open" (ie `rw_inherited_fifo_file_perms`.

If it really was a "#Leaked File Descriptors" then it wouldnt need the "open" permission provided by `rw_fifo_file_perms` in the first place.